src/network/messages/GameOperation.hpp

```diff
@@ -21,7 +21,7 @@ namespace spy::network::messages {
 
             GameOperation(util::UUID clientId, std::shared_ptr<gameplay::BaseOperation> operation);
 
-            [[nodiscard]] const std::shared_ptr<gameplay::BaseOperation> &getOperation() const;
+            [[nodiscard]] std::shared_ptr<const gameplay::BaseOperation> getOperation() const;
```

Comment:
Dann auch im Konstruktor und in der privaten Variable const ?

Reply:
done